plugins/osdn/ovs/controller.go

```diff
@@ -63,11 +69,16 @@ func alreadySetUp(multitenant bool, localSubnetGatewayCIDR string) bool {
 	}
 	found = false
 	for _, flow := range flows {
-		if !strings.Contains(flow, "table=3") {
+		if !strings.Contains(flow, STAMP_TABLE) {
+			continue
+		}
+		idx := strings.Index(flow, STAMP_ACTION)
+		if idx < 0 {
 			continue
 		}
-		if (multitenant && strings.Contains(flow, "NXM_NX_TUN_ID")) ||
-			(!multitenant && strings.Contains(flow, "goto_table:9")) {
```

Comment:
So, this is supposed to be checking if you switched from single-tenant to multi-tenant or vice versa. Except it doesn't actually work with the current version of the flows. Sigh.

Encode that into the note as well? `0.${STAMP}` for single-tenant rules, `1.${STAMP}` for multi-tenant?